sql/core/src/main/java/org/apache/spark/sql/sources/v2/vectorized/ColumnVector.java

```diff
@@ -14,32 +14,38 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.spark.sql.execution.vectorized;
+package org.apache.spark.sql.sources.v2.vectorized;
```

Comment:
Is v2 package proper for them? Are ColumnVector and related classes belong to data source v2 API?

Reply:
+1